nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/DynamicReloader.java

```diff
@@ -242,9 +248,22 @@ private void reloadApp(AppReloadInfo appInfo) throws IOException {
         deployParam.set(DeploymentProperties.PATH, appInfo.getApplicationDirectory().getCanonicalPath());
         deployParam.set(DeploymentProperties.NAME, appInfo.getApplication().getName());
         deployParam.set(DeploymentProperties.KEEP_REPOSITORY_DIRECTORY, "true");
+
+        Properties reloadFile = appInfo.readReloadFile();
+        boolean hotDeploy = Boolean.TRUE.toString().equals(reloadFile.get(DeployCommandParameters.ParameterNames.HOT_DEPLOY));
```

Comment:
Why not use `Boolean.parseBoolean`?
```suggestion
        boolean hotDeploy = Boolean.parseBoolean(reloadFile.get(DeployCommandParameters.ParameterNames.HOT_DEPLOY));
```